debian/sogo-openchange.postinst

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+case "$1" in
+    configure)
+        restart samba-ad-dc
+	;;
+
+    *)
+        echo "postinst called with unknown argument \`$1'" >&2
+        exit 0
+    ;;
+
+    abort-upgrade|abort-remove|abort-deconfigure)
```

Comment:
This will be never be called as `*` matches before with any argument, try by running the following command:

`bash -x debian/sogo-openchange.postinst abort-upgrade`